systers_portal/meetup/views.py

```diff
@@ -12,3 +12,17 @@ class MeetupLocationAboutView(MeetupLocationMixin, TemplateView):
 
     def get_meetup_location(self):
         return get_object_or_404(MeetupLocation, slug=self.kwargs['slug'])
+
+
+class MeetupView(MeetupLocationMixin, TemplateView):
```

Comment:
`slug_field` work when out model field is something else than `slug`, in our case we need ` slug_url_kwarg` that worked.

But I realized we need to check the returned meetup object should also be matched with requested meetup_location. So finally I found `get_queryset()` to be the correct option, here is the code pasted http://dpaste.com/1YHSBG3 but I am getting page not found error in browser while in terminal it is printing both the statements as expected. Do you have any thoughts for it?